compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -14082,6 +14092,7 @@ TR::Register *OMR::Z::TreeEvaluator::inlineVectorBinaryOp(TR::Node *node, TR::Co
                 "VFMAX/VFMIN is only supported on z14 onwards.");
             mask4 = getVectorElementSizeMask(node);
             breakInst = generateVRRcInstruction(cg, op, node, targetReg, sourceReg1, sourceReg2, 1, 0, mask4);
+            supportUnderMaskOperation = true;
             break;
         case TR::InstOpCode::VMX:
         case TR::InstOpCode::VMN:
```

Comment:
Am I missing something?  It looks like mask support isn't included for integer types with `vmmax` and `vmmin` operations.